doc/user-guide/tcs-user-guide-generic.rst

```diff
@@ -121,16 +140,27 @@ This is an often overlooked parameter when slewing to a target but it is also fu
 
    Illustration of the definition of position angle.
 
+With the advent of multiple-readout sections CCDs, defining the direction of the axis can be tricky.
+In general, it is assumed that the upper part of the CCD serial and parallel readout happens in the positive x and y-directions respectively.
+
+This is an often overlooked parameter when slewing to a target but it is also fundamental in determining the rotator/instrument orientation.
+
 Users can specify the position angle for the observation as well as use a couple different strategies for dealing with the rotator/instrument position.
 This is controlled using a pair of parameters; ``rot`` and ``rot_type``, which allows the user to specify the desired value (in degrees) and rotator strategy, respectively.
 
 The available strategies (and their meaning) are listed in :py:class:`RotType <lsst.ts.observatory.control.utils.RotType>`.
 By default ``rot_type = RotType.SkyAuto``, which means ``rot`` is treated as "position angle" and that it can be adjusted to be in range, if necessary.
 The adjustment consists of adding 180 degrees to the angle; this will usually, but not always, result in a rotation angle that is in range.
```

Comment:
why would it fail? is this a bug or do we need a sentence here explaining why and what to do?

Reply:
because of rotator range. I added a sentence.